Module4.1_Intro_to_Data_Structures_and_Algos/activities/Act4_Stacks/6.md

```diff
@@ -0,0 +1,56 @@
+<!--title={Using the Stack}-->
+
```

Comment:
A lot of this seems to be the same as 5.md, This cards should continue on the example by explaining how a program could be made to automatically solve the tower! Make sure it is connected to 5.md since its continuing with the same concept!